packages/scoutgame/src/points/calculateBuilderLevel.ts

```diff
@@ -39,6 +50,10 @@ export async function calculateBuilderLevels({
       },
       builderEvent: {
         season,
+        week: {
+          in: allSeasonWeeks
```

Comment:
instead of calculating all weeks, just do "not this week" or "not greater or equal to this week".